### PR TITLE
Port to ROS2

### PR DESCRIPTION
**Pull request by henningkayser**
Rebase and retarget of #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# object_recognition_msgs
+### This fork is part of Acutronic Robotics' efforts in the migration of Moveit! to ROS 2.0
```
Comment (jonbinney) [inline README.md]: Could you update this comment, since this is the main object_recognition_msgs repo, and not a fork?

**Review by jonbinney — APPROVED** [review]
One small comment; once that is addressed I'll merge and release.

File: README.md (hunk 1 above)
Comment (henningkayser) [reply README.md]: Of course! I just removed the added README as the other branches don't have one either.

**Comment by henningkayser** [comment]
> One small comment; once that is addressed I'll merge and release.

@jonbinney friendly ping

**Comment by jonbinney** [comment]
@henningkayser thanks for the reminder! Merged and released to foxy: https://github.com/ros/rosdistro/pull/25916

**Pull request merged**